crossing-training.lic

```diff
@@ -413,13 +413,22 @@ class CrossingTraining
   end
 
   def research
+    if @settings.cyclic_research
+      research_spell = @settings.crossing_cyclic_research_spells([])
+                       .min_by { |s| DRSkill.getxp(s) }
+      cyclic_research(research_spell)
```

Comment:
This appears to work for non-cyclic spells too. Is that intentional? If so, should any of the naming change?